### PR TITLE
feat(storage/benchmark): a script to plot results

### PR DESCRIPTION
**Pull request by coryan**
A new script to generate plots from the throughput_vs_cpu benchmark.
Took the time to write a README file for this directory, describing how
to run this benchmark and its script.

Part of the work for #4273 

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/4289)

### CODE REVIEW

File: google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# Copyright 2019 Google LLC
```
Comment (devjgm) [inline google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py:2]: Should this copyright year be updated?

File: google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+"""Summarize the results from running storage_throughput_vs_cpu_benchmark."""
+
+# %%
+import argparse
+import pandas as pd
+import plotnine as p9
+
+
+# %%
```
Comment (devjgm) [inline google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py:24]: What are these `# %%` comments?

File: google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py (hunk 1 above)
Comment (coryan) [reply google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py]: Fixed.

File: google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py (hunk 2 above)
Comment (coryan) [reply google/cloud/storage/benchmarks/storage_throughput_vs_cpu_plots.py:28]: They make it easier to use the script inside `Spyder`: https://www.spyder-ide.org/  I added a comment explaining them.

**Pull request merged**